05_testing_and_ci/intro_slides.md

```diff
@@ -0,0 +1,142 @@
+---
+type: slide
+slideOptions:
+  transition: slide
+  width: 1400
+  height: 900
+  margin: 0.1
+---
+
+<style>
+  .reveal strong {
+    font-weight: bold;
+    color: orange;
+  }
+  .reveal p {
+    text-align: left;
+  }
+  .reveal section h1 {
+    color: orange;
+  }
+  .reveal section h2 {
+    color: orange;
+  }
+</style>
+
+# Introduction to Testing
+
+---
+
+## Learning goals of the chapter
```

Comment:
@ajaust and me always (most of the time) used title case for slides headings ... we need a style guide eventually :D